### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by tatymolys**
SUB-6838: Update docs

---
PR Title: [SUB-6838] \<your PR title\>
---
### Description (optional)
Adding new chromatic and storybook documentation

Chromatic Documentation: https://submittable.atlassian.net/wiki/spaces/EN/pages/933134350/Guide+Chromatic+Workflow

### Changes
* List describing the changes your PR includes

### Testing
1. List of steps to test this PR

### Demo (recommended)
Screenshots or videos of the UI added in this PR

[SUB-6838]: https://submittable.atlassian.net/browse/SUB-6838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by wlaine-submittable** [comment]
Are you sure you want to include the logs in git? Otherwise approved.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-
 ## Live Storybook
-[Storybook](https://main--62f2ac6440da3214a498dc36.chromatic.com/) 
 
+[Storybook](https://main--62f2ac6440da3214a498dc36.chromatic.com/)
```
Comment (chan-fai-leung) [inline README.md:3]: add a line so it breaks storybook chromatic 

Storybook

Chromatic

Comment (chan-fai-leung) [reply README.md:3]: Screen Shot 2022-08-16 at 8 03 34 AM

Comment (tatymolys) [reply README.md:3]: Updated to give it its own section

**Review by chan-fai-leung — APPROVED** [review]

**Pull request merged**